posts/Yoann/2019-11-08-hibernate-search-quarkus.adoc

```diff
@@ -0,0 +1,1428 @@
+= Hibernate Search in Quarkus
+Yoann Rodière
+:awestruct-tags: [ "Hibernate Search", "Elasticsearch", "Quarkus" ]
+:awestruct-layout: blog-post
+:hsearchDocUrl: https://docs.jboss.org/hibernate/search/6.0/reference/en-US/html_single/
+:demoRepoUrl: https://github.com/hibernate/hibernate-demos/tree/master/hibernate-search/hsearch-quarkus
+---
+
+Hibernate libraries can be used with many frameworks,
+and we're striving to make sure that everyone gets their fair share of Hibernate goodness.
+So with the recent announce of https://quarkus.io/blog/announcing-quarkus-1-0/[the first release candidate of Quarkus],
+the "supersonic, subatomic" Java stack,
+it's worth mentioning that Hibernate ORM, Search and Validator are already included in Quarkus.
+
+Hibernate Search in particular, the library adding the power of full-text search to your ORM-based application,
+is included in its most recent version (6.0.0.Beta2), which adds first-class compatibility with Elasticsearch.
+
+Let's take this Quarkus release as an opportunity to have a closer look at Hibernate Search 6,
+and how it can be used in Quarkus.
+
++++<!-- more -->+++
+
+== Context
+
+=== What's Quarkus?
+
+https://quarkus.io/[Quarkus] is a Java stack for developing web applications.
+It integrates many Java libraries and provides tooling to ease development.
+Quarkus-based applications can run in a traditional HotSpot setup,
+but also (via https://www.graalvm.org/[GraalVM])
+as native binaries with a tiny memory footprint and impressively fast boot time.
```

Comment:
I think I would separate the two sentences: it's important that people understand that Quarkus have outstanding benefits with HotSpot too.